src/common/lib/utils.js

```diff
@@ -61,6 +62,17 @@ const utils = {
   stringToUint8Array: (str) => {
     return Uint8Array.from(str, (x) => x.charCodeAt(0));
   },
+  publishPaymentNotification: (message, paymentRequestDetails, response) => {
```

Comment:
I am not so happy with this being in `utils` this is very payment related and we need to make sure utils does not become too big with random functions. :D 

(but only for the future I think)